.github/workflows/update-snyk.yaml

```diff
@@ -1,5 +1,6 @@
 name: Snyk report update
 on:
+  workflow_dispatch: {}
   schedule:
     - cron: '0 0 * * 0' # midnight every Sunday
```

Comment:
FYI, GitHub cron times are UTC.  This cronjob would be equivalent to Saturday 5pm PST.  This actually works out great for me.

Reply:
Haha yeah actually works pretty well for me too